src/modules/UI/selectors.js

```diff
@@ -45,6 +45,26 @@ export const getArchivedWalletIds = (state: State): Array<string> => {
   return archivedWalletIds
 }
 
+export const getWalletLoadingPercent = (state: State) => {
```

Comment:
let me know if you want to try writing a test for this

Reply:
Done